### PR TITLE
Vagrant reloaded

### PR DESCRIPTION
**Pull request by lchrusciel**
Reloaded version of Vagrant, that should work with Sylius standard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,26 +12,29 @@ This configuration includes following software:
 
 # Usage
 
-First you need to create vagrant VM
+First you need to download a [Sylius standard project](https://github.com/Sylius/Sylius-Standard):
+```bash
+$ composer create-project -s beta -n sylius/sylius-standard acme
+```
 
+Then clone this repository inside newly created project
```
Comment (pamil) [inline README.md]: Missing `:` at the end :taco:

File: README.md [hunk 2]
```diff
@@ -12,26 +12,29 @@ This configuration includes following software:
...
+```
 
+Then clone this repository inside newly created project 
+```bash
+$ cd acme
+$ git clone git@github.com:Sylius/Vagrant.git vagrant
 ```
-$ cd etc/vagrant
+
+And build a vagrant:
```
Comment (pamil) [inline README.md]: `And run Vagrant:`?

File: README.md [hunk 3]
```diff
@@ -12,26 +12,29 @@ This configuration includes following software:
...
+```bash
+$ cd vagrant
 $ vagrant up
-```
+```
```
Comment (pamil) [inline README.md]: Why? :taco:

**Comment by pjedrzejewski** [comment]
Thanks Łukasz! :)

**Comment by pix-art** [comment]
Thanks @lchrusciel Been busy lately so didn't have time to keep up with the changes. Good to see other people use this aswell :-)

**Comment by lchrusciel** [comment]
@pix-art It was pretty easy, as the heavy-lifting part has been created already ;) We want this box to be one of our basic setups. But it would be nice to test it some how. I haven't checked it yet, but maybe you know some ci server, when we could build vagrant continuously and run some test scenarios in it? 

Anyway, thanks that you have prepared it before :)

**Comment by pix-art** [comment]
I've been looking at this myself when we switched this to a new repo, so far all i've understood is Travis wont be supporting it

> solarce commented on 24 May 2016
We do not support running VirtualBox inside any of our compute environments and currently have no plans to support it. We have some users who launch DigitalOcean droplets during their builds, and run virtualbox inside the Droplet, but this does result in the user needing to pay for the droplet costs.

They have a suggestion of people spinning up a Digital Ocean server and running it on there, and so on, but than you have a cost for the server and so on. So I think for now we have no way to test this.

**Pull request merged**